WordPress/Classes/ViewRelated/Post/Prepublishing/PrepublishingViewController.swift

```diff
@@ -680,6 +691,22 @@ private extension PrepublishingOption {
     }
 }
 
+// MARK: - Notifications
+extension PrepublishingViewController {
+    private func postCoordinatorDidUpdate(_ notification: Foundation.Notification) {
+        guard let updatedObjects = (notification.userInfo?[NSUpdatedObjectsKey] as? Set<NSManagedObject>) else {
```

Comment:
Could you please clarify the intention behind adding this observer?

Reply:
Added a clarifying comment b75e13d

This handles the case when you try to publish a draft post with a conflict.